components/prism-cmake.js

```diff
@@ -0,0 +1,8 @@
AKE_XCODE_SCHEME_DISABLE_MAIN_THREAD_CHECKER|CMAKE_XCODE_SCHEME_DYNAMIC_LIBRARY_LOADS|CMAKE_XCODE_SCHEME_DYNAMIC_LINKER_API_USAGE|CMAKE_XCODE_SCHEME_GUARD_MALLOC|CMAKE_XCODE_SCHEME_MAIN_THREAD_CHECKER_STOP|CMAKE_XCODE_SCHEME_MALLOC_GUARD_EDGES|CMAKE_XCODE_SCHEME_MALLOC_SCRIBBLE|CMAKE_XCODE_SCHEME_MALLOC_STACK|CMAKE_XCODE_SCHEME_THREAD_SANITIZER|CMAKE_XCODE_SCHEME_THREAD_SANITIZER_STOP|CMAKE_XCODE_SCHEME_UNDEFINED_BEHAVIOUR_SANITIZER|CMAKE_XCODE_SCHEME_UNDEFINED_BEHAVIOUR_SANITIZER_STOP|CMAKE_XCODE_SCHEME_ZOMBIE_OBJECTS|CPACK_ABSOLUTE_DESTINATION_FILES|CPACK_COMPONENT_INCLUDE_TOPLEVEL_DIRECTORY|CPACK_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION|CPACK_INCLUDE_TOPLEVEL_DIRECTORY|CPACK_INSTALL_DEFAULT_DIRECTORY_PERMISSIONS|CPACK_INSTALL_SCRIPT|CPACK_PACKAGING_INSTALL_PREFIX|CPACK_SET_DESTDIR|CPACK_WARN_ON_ABSOLUTE_INSTALL_DESTINATION|CTEST_BINARY_DIRECTORY|CTEST_BUILD_COMMAND|CTEST_BUILD_NAME|CTEST_BZR_COMMAND|CTEST_BZR_UPDATE_OPTIONS|CTEST_CHANGE_ID|CTEST_CHECKOUT_COMMAND|CTEST_CONFIGURATION_TYPE|CTEST_CONFIGURE_COMMAND|CTEST_COVERAGE_COMMAND|CTEST_COVERAGE_EXTRA_FLAGS|CTEST_CURL_OPTIONS|CTEST_CUSTOM_COVERAGE_EXCLUDE|CTEST_CUSTOM_ERROR_EXCEPTION|CTEST_CUSTOM_ERROR_MATCH|CTEST_CUSTOM_ERROR_POST_CONTEXT|CTEST_CUSTOM_ERROR_PRE_CONTEXT|CTEST_CUSTOM_MAXIMUM_FAILED_TEST_OUTPUT_SIZE|CTEST_CUSTOM_MAXIMUM_NUMBER_OF_ERRORS|CTEST_CUSTOM_MAXIMUM_NUMBER_OF_WARNINGS|CTEST_CUSTOM_MAXIMUM_PASSED_TEST_OUTPUT_SIZE|CTEST_CUSTOM_MEMCHECK_IGNORE|CTEST_CUSTOM_POST_MEMCHECK|CTEST_CUSTOM_POST_TEST|CTEST_CUSTOM_PRE_MEMCHECK|CTEST_CUSTOM_PRE_TEST|CTEST_CUSTOM_TESTS_IGNORE|CTEST_CUSTOM_WARNING_EXCEPTION|CTEST_CUSTOM_WARNING_MATCH|CTEST_CVS_CHECKOUT|CTEST_CVS_COMMAND|CTEST_CVS_UPDATE_OPTIONS|CTEST_DROP_LOCATION|CTEST_DROP_METHOD|CTEST_DROP_SITE|CTEST_DROP_SITE_CDASH|CTEST_DROP_SITE_PASSWORD|CTEST_DROP_SITE_USER|CTEST_EXTRA_COVERAGE_GLOB|CTEST_GIT_COMMAND|CTEST_GIT_INIT_SUBMODULES|CTEST_GIT_UPDATE_CUSTOM|CTEST_GIT_UPDATE_OPTIONS|CTEST_HG_COMMAND|CTEST_HG_UPDATE_OPTIONS|CTEST_LABELS_FOR_SUBPROJECTS|CTEST_MEMORYCHECK_COMMAND|CTEST_MEMORYCHECK_COMMAND_OPTIONS|CTEST_MEMORYCHECK_SANITIZER_OPTIONS|CTEST_MEMORYCHECK_SUPPRESSIONS_FILE|CTEST_MEMORYCHECK_TYPE|CTEST_NIGHTLY_START_TIME|CTEST_P4_CLIENT|CTEST_P4_COMMAND|CTEST_P4_OPTIONS|CTEST_P4_UPDATE_OPTIONS|CTEST_RUN_CURRENT_SCRIPT|CTEST_SCP_COMMAND|CTEST_SITE|CTEST_SOURCE_DIRECTORY|CTEST_SUBMIT_URL|CTEST_SVN_COMMAND|CTEST_SVN_OPTIONS|CTEST_SVN_UPDATE_OPTIONS|CTEST_TEST_LOAD|CTEST_TEST_TIMEOUT|CTEST_TRIGGER_SITE|CTEST_UPDATE_COMMAND|CTEST_UPDATE_OPTIONS|CTEST_UPDATE_VERSION_ONLY|CTEST_USE_LAUNCHERS|CYGWIN|ENV|EXECUTABLE_OUTPUT_PATH|GHS-MULTI|IOS|LIBRARY_OUTPUT_PATH|MINGW|MSVC|MSVC10|MSVC11|MSVC12|MSVC14|MSVC60|MSVC70|MSVC71|MSVC80|MSVC90|MSVC_IDE|MSVC_TOOLSET_VERSION|MSVC_VERSION|MSYS|PROJECT_BINARY_DIR|PROJECT_DESCRIPTION|PROJECT_HOMEPAGE_URL|PROJECT_NAME|PROJECT_SOURCE_DIR|PROJECT_VERSION|PROJECT_VERSION_MAJOR|PROJECT_VERSION_MINOR|PROJECT_VERSION_PATCH|PROJECT_VERSION_TWEAK|UNIX|WIN32|WINCE|WINDOWS_PHONE|WINDOWS_STORE|XCODE|XCODE_VERSION)\b/,
```

Comment:
We use single quotes for all our token names.

Reply:
Sure thing.